syncback/README.md

```diff
@@ -2,14 +2,95 @@
 
 This extension helps you sync files/directories from your container back to your local FS. Under the hood, it uses [`rsync`](https://rsync.samba.org/) to be smart about what files to transfer.
 
-## Usage
-tk
+## Overview
+This extension provides a function `syncback` which creates a manually-triggered local resource to sync files from a Kubernetes container back to your local filesystem. You can configure the source pod/container/directory/files+directories, and the destination location, as well as some other settings.
 
-## Common Use Cases
-tk
+Whenever you want to sync files back from the pod to your local filesystem, invoke the local resource from the Web UI:
 
-## Advanced Usage
-tk
+![A syncback resource in the sidebar, with the trigger button highlighted](assets/trigger-syncback.png)
+
+### Usage
+First, **make sure you have an up-to-date `rsync` on your local machine** (version >= 3.0.0).
```

Comment:
i wonder if the extension should check if `rsync` exists on load...

Reply:
way ahead of you! https://github.com/tilt-dev/tilt-extensions/pull/113